docs/source/conf.py

```diff
@@ -0,0 +1,20 @@
+# Configuration file for the Sphinx documentation builder.
+
+# -- Project information
+project = "datajudge"
+copyright = "(C) 2020-2021 QuantCo Inc."
+author = "QuantCo Inc."
+
+release = "1.0"
```

Comment:
Just out of curiosity, what's the difference between release and version?

Reply:
> version 
> The major project version, used as the replacement for |version|. For example, for the Python documentation, this may be something like 2.6.
> release
> The full project version, used as the replacement for |release| and e.g. in the HTML templates. For example, for the Python documentation, this may be something like 2.6.0rc1.
> If you don’t need the separation provided between version and release, just set them both to the same value.

https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-version